dataprep/clean/clean_duplication_utils.py

```diff
@@ -70,20 +78,22 @@ def _key_collision_cluster(self, cluster_method: str) -> None:
         }
         key_func = key_funcs[cluster_method]
         col = self._col
-        # get the count of each item in the dataframe and remove duplicates
-        counts = self._df[col].value_counts(sort=False)
-        no_dups = self._df.drop_duplicates(subset=[col]).reset_index()
-        # create a column "vals" containing tuples of the form (item, count)
-        no_dups.loc[:, "vals"] = no_dups[col].map(lambda val: (val, counts.loc[val]))
+        df = self._df.drop_duplicates(subset=[col])
```

Comment:
It looks like you're trying to get all of the unique values in column `col` here in a df. If so, what do you think about something like this `df = self._df[col].unique().to_frame(name=col)`?